proto/src/rr/dns_class.rs

```diff
@@ -46,39 +37,39 @@ impl DNSClass {
     /// Convert from &str to DNSClass
     ///
     /// ```
-    /// use trust_dns::rr::dns_class::DNSClass;
+    /// use trust_dns_proto::rr::dns_class::DNSClass;
     ///
     /// let var: DNSClass = DNSClass::from_str("IN").unwrap();
     /// assert_eq!(DNSClass::IN, var);
     /// ```
-    pub fn from_str(str: &str) -> DecodeResult<Self> {
+    pub fn from_str(str: &str) -> ProtoResult<Self> {
```

Comment:
I think `from_str()` is only used for the parsing of server configuration files? If so it can stay in `client`, I think. (I guess there are probably many things like this.)

Reply:
Yes... I'm trying to get things working before I go and break it. All the txt serialization will go into the `client`, but the impls all live with the `rdata`s.

This isn't compiling yet. I want to get everything back to compiling, then I'll do a bigger cleanup of moving this stuff around.